libpldmresponder/fru.cpp

```diff
@@ -22,118 +26,117 @@ namespace responder
 
 constexpr auto root = "/xyz/openbmc_project/inventory/";
 
-std::optional<pldm_entity>
-    FruImpl::getEntityByObjectPath(const dbus::InterfaceMap& intfMaps)
+pldm_entity FruImpl::getEntityByObjectPath(const dbus::ObjectValueTree& objects,
+                                           const std::string& path)
 {
-    for (const auto& intfMap : intfMaps)
+    pldm_entity entity{};
+
+    auto iter = objects.find(path);
```

Comment:
please use objects.contains(),

Reply:
I think we need to use find here because iter is used later

Reply:
got it , thanks